addons/elasticsearch/es-search-logs-mappings.json

```diff
@@ -0,0 +1,75 @@
+{
+  "mappings": {
+    "properties": {
+      "request.attributes": {
+        "type": "keyword"
+      },
+      "request.relationAttributes": {
+        "type": "keyword"
+      },
+
+      "userAgent": {
+        "type": "keyword"
+      },
+      "host": {
+        "type": "keyword"
+      },
+      "ipAddress": {
+        "type": "keyword"
+      },
+      "userName": {
+        "type": "keyword"
+      },
+
+
+      "entityGuidsAll": {
+        "type": "keyword"
+      },
+      "entityQFNamesAll": {
+        "type": "keyword"
+      },
+      "entityGuidsAllowed": {
+        "type": "keyword"
+      },
+      "entityQFNamesAllowed": {
+        "type": "keyword"
+      },
+      "entityGuidsDenied": {
+        "type": "keyword"
+      },
+      "entityQFNamesDenied": {
+        "type": "keyword"
+      },
+      "utmTags": {
+        "type": "keyword"
+      },
+
+      "hasResult": {
+        "type": "boolean"
+      },
+      "isFailed": {
+        "type": "boolean"
+      },
+
+      "errorDetails": {
+        "type": "text"
+      },
+      "errorCode": {
+        "type": "keyword"
+      },
+
+      "resultsCount": {
+        "type": "long"
+      },
+      "timestamp": {
```

Comment:
lets be specific here, what does `timestamp` represent ? is this creationTime or something else.

Reply:
This is creation time, will keep it same to be in sync with janusgraph_vertex_index's __timestamp